### ISSUE TITLE
#lang envy DSL

### ISSUE THREAD

**Issue by jackfirth** [lexi-lambda/envy#3]
(First off, this is really cool and I've wanted something like this for a while)

It would be especially nice if there was a `#lang envy`:

``` racket
#lang envy
aws-access-key-id
aws-secret-access-key
[foo-enabled? #:name "FOO_ENABLED" #:default #f]
```

Which would be equivalent to:

``` racket
#lang typed/racket/base
(require envy)
(define/provide-environment
  aws-access-key-id
  aws-secret-access-key
  [foo-enabled? #:name "FOO_ENABLED" #:default #f]
```

And it would be even nicer if it used syntax that cooperated with sweet expressions:

``` racket
#lang sweet-exp envy
aws-access-key-id
aws-secret-access-key
foo-enabled? #:name "FOO_ENABLED" #:default #f
```

**Comment by lexi-lambda** [comment]
Aha! You have anticipated my killer feature. Nice job. :wink: 

Anyway, yeah, I'm planning to implement this sooner rather than later. It shouldn't be too hard, and the only remaining question was how to get syntax like your `sweet-exp` example, but that's a perfect solution. Thanks for that!

**Comment by lexi-lambda** [comment]
Implemented in 06b4fee3abc756077ab0f277a0690a136d7c1496.
